layouts/_default/_markup/render-link.html

```diff
@@ -0,0 +1,2 @@
+<a href="{{ .Destination | safeURL }}"{{- with .Title }} title="{{ . }}"{{ end -}}>{{- with .Text | safeHTML }}{{ . }}{{ end -}}XX</a>
```

Comment:
I am sorry to retroactively re-review a PR but I was reviewing the [`safeHTMLAttr` docs for hugo](https://gohugo.io/functions/safe/htmlattr/) and they recommend that we not do this, and I see you use `printf` in a few ways like this recommends, should we be consistent?

```suggestion
<a {{ printf "href=%q" .Destination | safeHTMLAttr }}{{- with .Title }} title="{{ . }}"{{ end -}}>{{- with .Text | safeHTML }}{{ . }}{{ end -}}XX</a>
```